src/modules/auth/auth.module.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpModule } from "@nestjs/axios";
+import { Module } from "@nestjs/common";
+import { JwtModule } from "@nestjs/jwt";
+import { PassportModule } from "@nestjs/passport";
+import { UserModule } from "../user/module";
+import { AuthController } from "./entry-points/http/auth.controller";
+import { DiscordStrategy } from "./entry-points/http/discord.strategy";
+import { AuthService } from "./services/auth.service";
+import { configValues } from "../configuration";
+import { AppConfigModule } from "../configuration/configuration.module";
+
+@Module({
+  providers: [DiscordStrategy, AuthService],
+  controllers: [AuthController],
+  imports: [
+    PassportModule.register({}),
+    HttpModule,
+    UserModule,
+    JwtModule.register({
+      secret: configValues.auth.jwtSecret,
+      signOptions: { expiresIn: "7d" },
```

Comment:
Have we also considered using a refresh token pattern in addition to the access token we issue? It could be a bit over-engineered but I was wondering if you have thought about that.

Reply:
Didn't think about them in particular, but yeah, one disadvantage of the current implementation is the fact that the user will be logged out from the FE app when the token will expire. I can create a separate issue for it